roles/search/meta/argument_specs.yml

```
@@ -83,13 +83,23 @@ argument_specs:
       search_version:
         description: Alfresco Search Service version to use for deployment.
         required: false
-        default: 2.0.6
+        default: 2.0.14
       search_flavor:
         description: |
           Choose between alfresco-search-services and alfresco-insight-engine.
           Insight Engine requires a specific entitlement.
         required: false
         default: alfresco-search-services
+      search_zip_url:
+        description: |
+          The URL of the Search Service zip file to download.
+        required: false
+        type: str
+      search_zip_checksum_url:
+        description: |
+          The URL of the Search Service zip file checksum to download.
+        required: false
+        type: str
```

Wondering if we should add the defaults i here. More painful to maintain but provides more info to users

```suggestion
        type: str
        default: sha1:https://artifacts.alfresco.com/nexus/content/groups/public/org/alfresco/2.0.14/alfresco-search-services/2.0.14/alfresco-search-services-2.0.14.zip.sha1
```